roles/anaconda/tasks/main.yml

```diff
@@ -3,6 +3,7 @@
     url: "{{ anaconda_url }}"
     dest: "{{ anaconda_src }}"
     checksum: sha256:{{ anaconda_checksum }}
+    mode: "0755"
```

Comment:
The quotes are not needed when the 0 is included. https://docs.ansible.com/ansible/2.9/modules/file_module.html#parameter-mode I think it looks better without the quotes, but that's just a matter of taste :smile:

Reply:
Ok!